src/app/popup/styles.scss

```diff
@@ -0,0 +1,43 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font: inherit;
+}
+
+:root {
+  --space-025: 0.125rem; // 2px
+  --space-050: 0.25rem; // 4px
```

Comment:
i'd prefer having `--spacing-1: 4px` (or 8px) and all other spacings to be a product (`--spacing-2`, `--spacing-4` etc). imho it's simpler and easier to keep in mind